plugins/SegmentEditor/SegmentEditor.php

```diff
@@ -316,6 +316,16 @@ public static function isAddingSegmentsForAllWebsitesEnabled()
         return Config::getInstance()->General['allow_adding_segments_for_all_websites'] == 1;
     }
 
+    /**
+     * Returns whether create realtime segments is enabled or not.
+     *
+     * @return bool
+     */
+    public static function isCreateRealtimeSegmentsEnabled(): bool
+    {
+        return Config::getInstance()->General['enable_create_realtime_segments'] == 1;
+    }
```

Comment:
You could change this to use this new method as well:
https://github.com/matomo-org/matomo/blob/da44e3b3416f9012c774aa7abef19cb8cdbf58e1/plugins/SegmentEditor/API.php#L120-L121

Reply:
Thanks @sgiehl .

Sorry but I don't know what you mean. The lines you sent are not a method. Do you mean not checking for `== 1`? I thought for BC that is what we wanted.

Reply:
@geekdenz what @sgiehl is meant replacing that logic to use the new method like 

```
$realTimeSegmentsEnabled = SegmentEditor:: isCreateRealtimeSegmentsEnabled();
```

Not the other way around.